packages/ses/test/tame-legacy-regenerator-helper.test.js

```diff
@@ -0,0 +1,53 @@
+import test from 'ava';
+import '../index.js';
+
+lockdown({ legacyRegeneratorRuntimeTaming: 'unsafe-ignore' });
+
+test('lockdown Iterator.prototype[@@iterator] is tamed', t => {
+  const IteratorProto = Object.getPrototypeOf(
+    Object.getPrototypeOf([].values()),
+  );
+  const desc = Object.getOwnPropertyDescriptor(IteratorProto, Symbol.iterator);
+  if (!desc || !desc.get || !desc.set) throw new Error('unreachable');
+  t.is(desc.configurable || desc.enumerable, false);
+  t.is(desc.value, undefined);
+
+  const { get, set } = desc;
+  t.is(
+    Function.prototype.toString.call(get),
+    'function get() { [native code] }',
+  );
+  t.is(
+    Function.prototype.toString.call(set),
+    'function set() { [native code] }',
+  );
```

Comment:
```suggestion
  const { get } = desc;
```

Once you move  tameRegeneratorRuntime immediately below the call to enablePropertyOverrides as suggested above, these tests would no longer pass. At first I was about to suggest testing the new toString behavior, but then I remembered that ecma262 makes that optional and Moddable XS makes use of that to omit source strings from the compiled form. So I think you can just drop these. IIRC, we don't test the printed form of the getter and setters installed by enable-property-overrides.js, which would be the closest analogy.